### ISSUE TITLE
Many comments are not supported

### ISSUE THREAD

**Issue by gongph** [neogeek/doxdox#42]
Many comments are not supported，For example:

```js
/**
 *  @constant
 *  @default
 */
const RED = 0xff0000;
```

doxdox only render `Methods`, but `Members` is not render !

**Comment by neogeek** [comment]
This is good to keep in mind as I'm currently in the process of vetting other parsers for use with doxdox (currently shipping with [dox](https://github.com/tj/dox)). Might put together a comparison table before going all in with a new parser.

**Comment by agauniyal** [comment]
yes please add constant and default and other jsdoc comments too 😸. I've high hopes with this documentation generator and I'm already it 👍

**Comment by Elyx0** [comment]
Hi, any updates?

**Comment by yunnysunny** [comment]
The tags of `typedef` or `function` are not supported either.

**Comment by neogeek** [comment]
With the complete rewrite of this package, we have switched to using the JSDoc parser, so this is now possible. I will update this ticket when I make progress on this.
